index.html

```diff
@@ -25,17 +29,12 @@
 					<!-- logo container -->
 					<div class="logo-container">
 						<div class="img-container logo-icon">
-							<picture>
-								<source
-									media="(min-width:768px)"
-									srcset="./assets/icons/global/logo-icon-big.svg"
-								/>
-								<img
-									src="./assets/icons/global/logo-icon.svg"
-									alt="If These Lands Could Talk logo-icon"
-									style="width: auto"
-								/>
-							</picture>
+							<a href="/index.html">
+								<picture>
+								<source media="(min-width:1180px)" srcset="/assets/icons/global/logo-icon-big.svg"/>
+								<img src="/assets/icons/global/logo-icon.svg" alt="If These Lands Could Talk logo-icon" style="width: auto"/>
```

Comment:
One quick adjustment needed: noticed in tablet in mobile views the placeholder events need to be centered for the img and text

Currently:
![image](https://github.com/indigenousfriends/itlct-website-redesign/assets/98284615/0f652b41-cf55-4333-bc1c-50b4198cc828)

Figma Reference: 
![image](https://github.com/indigenousfriends/itlct-website-redesign/assets/98284615/2720209b-c6c5-4483-a81f-6aaf580fd66a)

For the img, i think the nested img needs to be centered inside `<picture>`

text just needs to be aligned

Reply:
resolved this issue